vita3k/host/include/host/functions.h

```diff
@@ -20,8 +20,12 @@
 #include <string>
 #include <vector>
 
+class KeyStore;
+
 struct SfoFile;
 
+void install_pup(std::string pup, std::string output);
```

Comment:
```suggestion
void install_pup(const std::string &pup, const std::string &output);
```

Use this to make string a reference, prevent unnecessary copy if you don't modify any of the parameter.